tools/deploy_translatable_strings.sh

```diff
@@ -14,74 +14,94 @@
 
 # Script for pushing the translatable messages to poBranch.
 
+# DO NOT `set -x`.  We have to pass secrets to `openssl` on the command line,
+# and we don't want them appearing in the log.  This script instead manually
+# 'echo's its status at various points.
+set -eu -o pipefail
+
+if [[ "$#" -ne 1 ]]; then
+    echo "Usage: deploy_translatable_string.sh /path/to/translations/artifact" >&2
+    exit 1
+fi
 
 # Variables used by this script.
 # From github actions the docs/locale/en directory from the sphinx build
 # gets downloaded from the github actions artifacts as deploy directory
-TARGET_REPOSITORY="git@github.com:qiskit-community/qiskit-translations.git"
-SOURCE_DIR=`pwd`
-SOURCE_LANG='en'
-
-SOURCE_REPOSITORY="git@github.com:Qiskit/qiskit.git"
-TARGET_BRANCH_PO="main"
-DOC_DIR_PO="deploy"
-TARGET_DOCS_DIR_PO="docs/locale"
-
-echo "show current dir: "
-pwd
-
-echo "Setup ssh keys"
-pwd
-set -e
-# Add poBranch push key to ssh-agent
-openssl enc -aes-256-cbc -d -in ../tools/github_poBranch_update_key.enc -out github_poBranch_deploy_key -K $encrypted_deploy_po_branch_key -iv $encrypted_deploy_po_branch_iv
-chmod 600 github_poBranch_deploy_key
-eval $(ssh-agent -s)
-ssh-add github_poBranch_deploy_key
-
-# Clone to the working repository for .po and pot files
-popd
```

Comment:
This `popd` was the first directory-stack operation in the file, so it'd be an error.